nss-c31-unusual-prestige/nutshell
# Time stamp

# Description

Each article now has an edit button that populates an edit form with a save button that updates data and refreshes dom when clicked.

Applies to issue # 4

## Type of change

Please delete options that are not relevant.




# Testing Instructions

pull down branch and run grunt.
Articles should display click edit button enter new article and save. it should post to the Dom.
This should be your main.js

import apiManager from "./apiManager.js";
import newsFeedDOM from "./newsFeed/newsDomManage.js"
import newsForm from "./newsFeed/newsForm.js"
import newsEventHandler from "./newsFeed/newsEventHandler.js"


newsForm.newsForm()
newsFeedDOM.childArticleContainer()
newsFeedDOM.listAllNews()



# Checklist:
